docs/source/gbd2019_models/risk_exposures/wasting_state_exposure/index.rst

```diff
@@ -0,0 +1,392 @@
+  3. 2x4 state model with 2 disease states and 4 wasting states with death and fertility (tbd)
+
+
+Assumptions and Limitations
++++++++++++++++++++++++++++
+
+Describe the clinical and mathematical assumptions made for this cause model,
+and the limitations these assumptions impose on the applicability of the
+model.
+
+Markov chains
+-------------
+
+.. todo::
+  add some detail about markov chains, define mathematic notations 
+
+Equilibirum
+-----------
+
+.. todo::
+  add some detail about equilirium
+
+Arborescence
+------------
+
+.. todo::
+  add some detail about graph theory and the process we did to discover the pattern in our markov chains
+
+As a rule for the finiate state machines, the numerator of the prevalence of a state is the sum of the product of all edges in every unique anti-arborescence (graph theory).
+
+.. note::
+  This section will become the methods section in the manuscript. 
+
+
+.. _2.1: 
+
+Restrictions
+------------
+
+.. list-table:: GBD 2019 Risk Exposure Restrictions
+   :widths: 15 15 20
+   :header-rows: 1
+
+   * - Restriction Type
+     - Value
+     - Notes
+   * - Male only
+     - False
+     -
+   * - Female only
+     - False
+     -
+   * - Age group start
+     - post-neonatal 1mo to 1 year, id 4 
+     - exclude neonatal age groups
+   * - Age group end
+     - 1yr to 4yr id 5
+     - 
+
+..	todo::
+
+	Determine if there's something analogous to "YLL/YLD only" for this section
+
+.. _2.2: 
+
+Risk Exposure Model Diagram
+---------------------------
+
+.. _2.2.1: 
+
+Finite state machine 1x4 
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+.. image:: wasting_state_1x4.svg
+
+To solve the 10 transition pobabilities, we use a Markov Chain transition matrix **T**. 
+
+T = 
+
+.. csv-table:: 
+   :file: wasting_state_1x4.csv
+   :widths: 5, 5, 5, 5, 5
+
+
+:math:`π_{T}` = 
+
++----+----+----+----+
+| p4 | p3 | p2 | p1 |
++----+----+----+----+
+
+:math:`π_{T}` is the eigenvector at equilibriuum
+
+  a) :math:`π_{T}\times\text{T} = π_{T}` (the T means transposed, this is a 1 row vector)
+  b) :math:`\sum_{\text{i=p}}` = :math:`π_{T}`
+  c) :math:`π_{i}` ≥ 0 , these are GBD 2019 age/sex/location/year-specific prevalence for wasting categories 1-4
+
+
+Solving a)
+
+  1)  p4xs4 + p3xr4 = p4 
+  2)  p4xi3 + p3xs3 + p2xr3 = p3
+  3)  p3xi2 + p2xs2 + p1xr2 = p2
+  4)  p2xi1 + p1xs1 = p1
+
+Rows of the P matrix sums to 1
+
+  5)  s4 + i3 =1
+  6)  r4 + s3 + i2 = 1
+  7)  r3 + s2 + i1 =1
+  8)  r2 + s1 = 1
+
+We have duration of treated and untreated sam and mam as well as coverage from the literature :   
+
+  9)  r2 = 1/Dsam   
+  10) r3 + i1  = 1/Dmam   
+
+where
+
+ - Duration of cat 1: Dsam = C x Dsam_tx + (1-C)Dsam_ux ~ 40 days stand in value (will refine)
+ - Duration of cat 2: Dmam = C x Dmam_tx + (1-C)Dmam_ux ~ 70 days stand in value (will refine)
+ - tx is treated
+ - ux is untreated
+ - C is treatment coverage proportion
+
+We solve for the unknowns using the matrix solution
```

Comment:
Somewhere should note that we solve for the solution for the first state model, but will need to find the closed form graph theory solution for models two and three.